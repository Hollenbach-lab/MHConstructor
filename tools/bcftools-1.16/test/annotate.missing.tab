chr1	10	A	T	.	.	.
chr1	20	A	T	new	0.9	9
chr1	30	A	T	new	0.9	9
chr1	40	A	T	new	0.9	9
chr1	50	A	T	.	.	.
