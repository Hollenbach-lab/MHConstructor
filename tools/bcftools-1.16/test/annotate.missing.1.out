##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##INFO=<ID=TFLT,Number=1,Type=Float,Description="Dummy float">
##INFO=<ID=TINT,Number=1,Type=Integer,Description="Dummy int">
##INFO=<ID=TSTR,Number=1,Type=String,Description="Dummy str">
##contig=<ID=chr1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	10	.	A	T	.	.	TFLT=0.1;TINT=1;TSTR=old
chr1	20	.	A	T	.	.	TFLT=0.9;TINT=9;TSTR=new
chr1	30	.	A	T	.	.	TFLT=0.9;TINT=9;TSTR=new
chr1	40	.	A	T	.	.	TSTR=new;TFLT=0.9;TINT=9
chr1	50	.	A	T	.	.	.
